apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultInfo/index.tsx

```diff
@@ -113,21 +129,24 @@ export function EvaluationResultInfo({
             {selectedTab === 'messages' && (
               <EvaluationResultMessages providerLog={providerLog} />
             )}
-            <div className='w-full flex justify-center'>
-              <Button variant='link' onClick={onClickOpen}>
-                Check original log
-                <Icon name='arrowRight' widthClass='w-4' heightClass='h-4' />
-              </Button>
-            </div>
+            {evaluationResult.evaluatedProviderLogId && (
```

Comment:
So if result have `evaluatedProviderLogId` it means it has a document log evaluated? 
Why not check the `documentLog` presence?

Reply:
we always evaluate provider logs, not document logs, so it's more accurate to only show the option to see the original log if there is an evaluated provider log

Reply:
I know it is an edge case, but how do evaluations work with manually uploaded logs by API? We don't have a provider log in that case do we? Only a document log? Or maybe I'm mixing things.

Reply:
we do have provider log for those also